bin/help.txt

```diff
@@ -11,7 +11,7 @@ Options:
       --highlight-theme <theme>               Highlight theme [default: zenburn]
       --css <files>                           CSS files to inject into the page
       --scripts <files>                       Scripts to inject into the page
-      --assets-dir <dirname>                  Defines assets directory name [default: _assets]
+      --assets-dir <dirname>                  Defines assets directory name [default: assets]
```

Comment:
Why this change? Might be a breaking change to some and it's a bit much for one PR.

Reply:
GitHub seems to change the folder name if there is a dash at the start of the name. See [Linking to a page with a dash (hyphen) in the title In GitHub wiki](https://stackoverflow.com/q/77244370/3416774)